tests/test_regression.py

```diff
@@ -0,0 +1,124 @@
+"""
+    Suite of tests that go through the entire process of creating a new fktable
+    from a empty folder.
+
+    The target theory is 400 and the relevant `.toml`, theory runcard and eko template
+    are downloaded from https://github.com/NNPDF/theories during this test so this tests
+    has the double effect of ensuring compatibility between both repositories.
+"""
+from pathlib import Path
+from subprocess import run
+from urllib.request import urlretrieve
+
+import numpy as np
+import pytest
+from pineappl.fk_table import FkTable
+from yaml import dump, safe_load
+
+THEORIES_REPO = "https://raw.githubusercontent.com/NNPDF/theories/main"
```

Comment:
What is the current plan about this repo? We gave up on storing EKOs, so what are we storing there?

Reply:
It stores anything necessary to run pineko and create new theories. So basically the template cards (and also the cards, because they are cheap). The theory cards, the yaml... and the grids. Only the grids will be "a reference to the grid" as we discussed to avoid storing big files in github but we know we will never do it.

Reply:
> but we know we will never do it.

I actually agree xD

But I believe that even the grids will be inconsistently stored (since you don't want to do it for any throwaway work-in-progress theory).

Concerning the others:
- we are approaching the point in which the yaml-db will be inside the new Commondata (slowly approaching...)
- the theory cards at some point should be an interface to the theory db, because we definitely want to be consistent with it (we just need a dedicated effort to make the interface, and maybe even improving the DB itself)

So, I feel like NNPDF/theories is a temporary solution (that might last quite a while, because it is not a priority to drop it), but that's fine :)